pom.xml

```diff
@@ -94,6 +101,12 @@
             <artifactId>liquibase-core</artifactId>
             <version>${liquibase.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
+            <version>1.30</version>
```

Comment:
CVE for snakeyaml 1.30 use at least 1.31, I believe 1.32 is the latest.
https://github.com/advisories/GHSA-3mc7-4q67-w48m